forge/csrc/passes/commute_utils.cpp

```diff
@@ -898,16 +898,17 @@ bool can_commute_past_op(
     return (is_elementwise(op) and op->op_name() != "interleave") or is_quantization_ops(op);
 }
 
-void update_reshape_attr(graphlib::OpNode *reshape, graphlib::Shape new_shape)
+void update_reshape_attr(graphlib::OpNode *op, graphlib::Shape new_reshape_shape)
 {
-    if (reshape->op_name() != "reshape")
```

Comment:
Why are we removing this check? At the end, we're changing op type to reshape. Are there some cases where we convert ops from one type to reshape?

Reply:
That check is required and it is added now.